### PR TITLE
Django 1.8 support, drop <1.6, code cleanup, fix admin actions + enab…

### PR DESCRIPTION
**Pull request by cyberstar**
Changes that were introduced:
- Django 1.8 support,
- drop support of Django versions <1.6
- code clean up
- fix admin actions
- enable admin actions by default.
- misc small improvements.

### CODE REVIEW

File: django_tablib/admin/__init__.py [hunk 1]
```diff
@@ -1,111 +1,132 @@
...
+        for export_format in self.formats:
+            if export_format not in mimetype_map:
+                raise ValueError(INVALID_EXPORT_FORMAT_ERROR_MSG.format(
+                    export_format, ', '.join(mimetype_map.keys())))
         super(TablibAdmin, self).__init__(*args, **kwargs)
 
+    def get_info(self):
+        """
+        Get model info in a form of (app_label, model_name).
+        """
```
Comment (intellectronica) [inline django_tablib/admin/__init__.py:48]: I could use a bit more of an explanation here. What's the difference between model and module? Why would you use one and not the other in each case?

File: django_tablib/datasets.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
-from __future__ import absolute_import
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals, absolute_import
```
Comment (intellectronica) [inline django_tablib/datasets.py]: alphabetical order, where possible

**Comment by intellectronica** [comment]
:+1:

File: django_tablib/admin/__init__.py (hunk 1 above)
Comment (cyberstar) [inline django_tablib/admin/__init__.py:48]: `module_name` is removed in `Django` 1.8, 1.7 produces deprecation warnings, but 1.6 doesn't have `model_name` (afaik), and to not to make same code twice i did a small helper function. Not sure that it should have such detailed info in its description...

Comment (intellectronica) [inline django_tablib/admin/__init__.py:48]: Great. Perhaps you could add that to the docstring?

**Pull request merged**